SPECS/sm.spec

```
@@ -422,8 +416,68 @@ Manager and some other packages
 %{_unitdir}/fairlock@.service
 %{_libexecdir}/fairlock
 
+%post fairlock
+## On upgrade, shut down existing lock services so new ones will
+## be started. There should be no locks held during upgrade operations
+## so this is safe.
+if [ $1 -gt 1 ];
+then
+    /usr/bin/systemctl list-units fairlock@* --all --no-legend | /usr/bin/cut -d' ' -f1 | while read service;
+    do
+        /usr/bin/systemctl stop "$service"
+    done
+fi
 
 %changelog
+* Thu Mar 20 2025 Ronan Abhamon <ronan.abhamon@vates.tech> - 3.2.12-3.1
+- Rebase on 3.2.12-3
+- Sync patches with our latest 3.2.12-8.3 branch
+- Remove patches merged upstream:
+- - 0028-CA-398425-correctly-check-for-multiple-targets-in-iS.patch
+- - 0030-fix-getAllocatedSize-is-incorrect-75.patch
+- - 0047-Define-and-details-attr-on-Failure-mock.patch
+- - 0064-feat-add-HPE-Nimble-multipath-configuration.patch
```

We're not quoting their changelog here.

(and anyway @ydirson convinced me to indent with spaces for easier diffs with the changelog from the XS-8.3 branch)

```suggestion
  - 0028-CA-398425-correctly-check-for-multiple-targets-in-iS.patch
  - 0030-fix-getAllocatedSize-is-incorrect-75.patch
  - 0047-Define-and-details-attr-on-Failure-mock.patch
  - 0064-feat-add-HPE-Nimble-multipath-configuration.patch
```